test/integration/apis/oracle.test.ts

```diff
@@ -27,16 +27,19 @@ describe("OracleAPI", () => {
         return api.disconnect();
     });
 
-    describe("getInfo", () => {
+    describe("Oracle", () => {
         it("should return oracle info", async () => {
             const info = await oracle.getInfo();
             assert.equal(info.names[0], "Bob");
             assert.isTrue(info.online);
             assert.equal(info.feed, "DOT/BTC");
         });
-    });
 
-    describe("setExchangeRate", () => {
+        it("should have a rate of 3855.23187", async () => {
```

Comment:
maybe name the test "docker-composeinitial setup should have a rate of X". Just so we are clear this would a failure in docker-compose if were to happen